adhocracy4/phases/models.py

```diff
@@ -20,6 +20,29 @@ def active_phases(self):
     def finished_phases(self):
         return self.filter(end_date__lte=timezone.now())
 
+    def past_phases(self):
```

Comment:
I am not sure, but maybe we missed something: Do we have a problem once start/end date become optional due to the dashboard overhaul?